src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FlowPathModel.groovy

```diff
@@ -0,0 +1,29 @@
+package org.openkilda.functionaltests.helpers.model
```

Comment:
It becomes quite crowded in this package. I'd recommend to add new package, e.g. `path`.

Reply:
Done.